lang/java/reef-io/src/main/java/org/apache/reef/io/network/naming/serialization/NamingLookupRequestCodec.java

```diff
@@ -52,7 +52,7 @@ public NamingLookupRequestCodec(final IdentifierFactory factory) {
    */
   @Override
   public byte[] encode(final NamingLookupRequest obj) {
-    final List<CharSequence> ids = new ArrayList<>();
+    final List<String> ids = new ArrayList<>();
```

Comment:
This change was due to using `String` for the string type in Avro. I reverted this change since we don't use `String` anymore.